src/main/java/com/gluonhq/substrate/target/AbstractTargetConfiguration.java

```diff
@@ -244,4 +254,34 @@ public boolean runUntilEnd(Path appPath, String appName) throws IOException, Int
         return true;
     }
 
+    // Default settings below, can be overridden by subclasses
+
+    String getAdditionalSourceFileLocation() {
+        return "/native/linux/";
```

Comment:
This seems to be a constant.  Do we need a method?

Reply:
Yes, as it is overridden in e.g. IosTargetConfiguration